pipeline.py

```diff
@@ -240,13 +252,16 @@ def ilab_pipeline(
     model_source_task = dsl.importer(
         artifact_uri=sdg_base_model, artifact_class=dsl.Model
     )
+    model_source_task.after(prerequisites_check_task)
+    model_tokenizer_source_task.after(prerequisites_check_task)
```

Comment:
Optional: It'd be nice if this line was directly below `model_tokenizer_source_task = dsl.importer(` since it makes the code easier to read.